src/integrationTest/resources/application.yaml

```diff
@@ -51,6 +51,9 @@ spring:
   main:
     allow-circular-references: true
     allow-bean-definition-overriding: true
+  jackson:
+    generator:
+      auto-close-json-content: false
```

Comment:
does it need to be on both the yaml and the servicecontextconfig? I would have thought one would do it

Reply:
ha yeah, I thought so too, but lots of tests seem to just create a fresh objectmapper and throw it away at the end of the test, so needed to add it to both to get the tests to pass (imagine we should refactor them later at _some_ point)